src/utils/TestingStorage.tsx

```diff
@@ -3,27 +3,39 @@ import React from 'react';
 import {Button} from 'react-native-paper';
 import {
   getAllKeysConver,
-  getData,
-  removeConversation,
-  storeData,
+  getDataLocal,
+  removeConversationLocal,
+  storeDataLocal,
   clearAll,
 } from '../controllers/localStorage';
 import {contactsList} from './constants';
+import styled from 'styled-components/native';
+import {getUserByEmail} from '../controllers/firebaseFirestore';
+
+const ButtonTrigger = styled.TouchableOpacity`
+  height: 100px;
+  width: 100px;
+  background-color: #bb4550;
+  border-radius: 50px;
+  align-items: center;
+  justify-content: center;
+  align-self: center;
+`;
 
 const TestingStorage = () => {
   const handlePressSave = () => {
     console.log('saved');
-    const algo = storeData('contacts', contactsList);
+    const algo = storeDataLocal('contacts', contactsList);
     console.log(algo);
   };
   const handlePressGet = async () => {
-    const algo = await getData('contacts');
+    const algo = await getDataLocal('contactList');
     console.log(algo);
   };
 
   const handlePressDel = async () => {
     console.log('got');
-    const algo = await removeConversation('128');
+    const algo = await removeConversationLocal('128');
```

Comment:
HARDCODED 128, better to move it to a const while the function should recieve that as argument